tchannel/response.py

```diff
@@ -74,17 +75,31 @@ class TransportHeaders(object):
         'scheme',
     )
 
-    def __init__(self,
-                 failure_domain=None,
-                 scheme=None,
-                 **kwargs):
```

Comment:
No, it's not breaking. This was in place because `to_kwargs` would send in a dict with `None` values for headers that were not valid transport headers for response but were valid for request.

Since `from_dict` is now responsible for instantiating `TransportHeaders` you will never get unexpected kwargs there.